specs/eip4844/polynomial-commitments.md

```diff
@@ -336,35 +353,41 @@ def blob_to_kzg_commitment(blob: Blob) -> KZGCommitment:
 #### `verify_kzg_proof`
 
 ```python
-def verify_kzg_proof(polynomial_kzg: KZGCommitment,
+def verify_kzg_proof(commitment: Bytes48,
                      z: Bytes32,
                      y: Bytes32,
-                     kzg_proof: KZGProof) -> bool:
+                     proof: Bytes48) -> bool:
     """
     Verify KZG proof that ``p(z) == y`` where ``p(z)`` is the polynomial represented by ``polynomial_kzg``.
     Receives inputs as bytes.
     Public method.
     """
-    return verify_kzg_proof_impl(polynomial_kzg, bytes_to_bls_field(z), bytes_to_bls_field(y), kzg_proof)
+    validate_kzg_g1(commitment)
+    validate_kzg_g1(proof)
+
+    return verify_kzg_proof_impl(commitment,
```

Comment:
This needs to be cast explicitly to match the expected types of `_impl`

Reply:
Pushed 4204f4c2d1eed07b52af4393c9b20eaeb952e60f which introduces explicit type conversion between "untrusted" bytes and "trusted" BLS types.